### PR TITLE
FAQ updated with new ticket cancellation and transfer policies

### LINKED ISSUE #84: [URGENT] Update ticket FAQ
Update ticket FAQ from this information:

Policy for ticket transfer & cancellation:

- All tickets are transferable.

- Early bird tickets cannot be canceled. They can only be transferred.

- All other tickets can be canceled till 10th Oct 2017.

- Tickets can be transferred till 10th Oct 2017 using the ticket transfer link .

### PR DESCRIPTION
**Pull request by realslimshanky**
this fixes #84 
can be tested on https://realslimshanky.github.io/inpycon2017/faqs.html

### CODE REVIEW

**Comment by CuriousLearner** [comment]
Looks good so far, please add about how the dev sprint entry works, on what ticket basis; etc.

File: templates/faqs.html [hunk 1]
```diff
@@ -141,8 +141,10 @@
 											<h4><span>Q.</span>I need to cancel my ticket due to unavoidable cicumstances. What is the refund policy?</h4>
 											<div class="tg-panelcontent">
 												<div class="tg-description">
-													<p><span>A.</span>We allow tickets to be cancelled, but cancellation might be barred in the last few days leading up to the conference. This space will be updated if cancellation is barred at any time.
-													</p>
+												<p><span>A.</span>- All tickets are transferable.<br>
+														&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;- Early bird tickets cannot be canceled. They can only be transferred.<br>
```
Comment (SaptakS) [inline templates/faqs.html]: Why use `&nbsp;` and not put the content inside a div and give the div margin/padding?

Comment (sayanchowdhury) [reply templates/faqs.html]: :+1:

Comment (realslimshanky) [reply templates/faqs.html]: I followed the format used on the page above. Let me try a better solution.

Comment (CuriousLearner) [reply templates/faqs.html]: Can we please get this in by EOD today?

Comment (realslimshanky) [reply templates/faqs.html]: I'm fixing it right now.

**Comment by realslimshanky** [comment]
I'm done with the changes. Please review.
Changes can also be tested on [this](https://realslimshanky.github.io/inpycon2017/faqs.html) link.

**Comment by CuriousLearner** [comment]
@SaptakS @sayanchowdhury Please review this.

File: faqs.html [hunk 2]
```diff
@@ -225,11 +224,21 @@
...
+												<p class="tg-offset">- All other tickets can be canceled till 10th Oct 2017.</p>
+												<p class="tg-offset">- Tickets can be transferred till 10th Oct 2017 using the ticket transfer <a href="https://www.explara.com/manage-tickets">link</a>.</p>
+												</div>
+											</div>
+										</div>
+										<div class="tg-panel">
+											<h4><span>Q.</span>Do I have to buy a seperate ticket for the Dev Sprint?</h4>
+											<div class="tg-panelcontent">
+												<div class="tg-description">
+													<p><span>A.</span>No, you don't have to buy a seperate ticket for Dev Sprint. But you'll need to have either a Conference Ticket or a Workshop Ticket to be able to enter Dev Sprint.</a></p>
```
Comment (CuriousLearner) [inline faqs.html]: Also, add here that due to space constraint, entry to Dev Sprints will be on first-come first-serve basis

File: faqs.html [hunk 3]
```diff
@@ -171,19 +171,18 @@
...
 										<div class="tg-panel">
 											<h4><span>Q.</span>What all does the Conference ticket (November 4 &amp; 5) include? </h4>
 											<div class="tg-panelcontent">
 												<div class="tg-description">
-													<p><span>A.</span>- Full conference pass for two days.<br>
-													&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;- Breakfast, Lunch and Hi-Tea.<br>
-													&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;- PyCon India T-shirt which will be distributed on Sep 25.
-													</p>
+													<p><span>A.</span>- Full conference pass for two days.</p>
+													<p class="tg-offset">- Breakfast, Lunch and Hi-Tea.<br>
```
Comment (sayanchowdhury) [inline faqs.html]: High Tea

**Comment by SaptakS** [comment]
Looks good to me.

File: faqs.html (hunk 2 above)
Comment (sayanchowdhury) [inline faqs.html]: You should mention that there is limited space, so FCFS?

**Review by sayanchowdhury — CHANGES_REQUESTED** [review]

**Comment by sayanchowdhury** [comment]
I have two suggestions other than that it looks good to merge. :cake:

**Comment by realslimshanky** [comment]
@sayanchowdhury I've made changes according to your review.

**Review by sayanchowdhury — APPROVED** [review]

**Comment by sayanchowdhury** [comment]
:taco:

**Review by CuriousLearner — APPROVED** [review]

File: faqs.html [hunk 4]
```diff
@@ -171,19 +171,18 @@
...
 											<h4><span>Q.</span>What all does the Conference ticket (November 4 &amp; 5) include? </h4>
 											<div class="tg-panelcontent">
 												<div class="tg-description">
-													<p><span>A.</span>- Full conference pass for two days.<br>
-													&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;- Breakfast, Lunch and Hi-Tea.<br>
-													&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;- PyCon India T-shirt which will be distributed on Sep 25.
-													</p>
+													<p><span>A.</span>- Full conference pass for two days.</p>
+													<p class="tg-offset">- Breakfast, Lunch and High-Tea.<br>
+													   - PyCon India T-shirt which will be distributed on Sep 25.</p>
```
Comment (CuriousLearner) [inline faqs.html]: This date should be changed to Nov 4 or removed entirely for now.

**Review by CuriousLearner — CHANGES_REQUESTED** [review]

File: faqs.html (hunk 4 above)
Comment (realslimshanky) [reply faqs.html]: Sure. Changing to 4th.

**Comment by realslimshanky** [comment]
@CuriousLearner date has been fixed.

**Review by CuriousLearner — APPROVED** [review]

**Pull request merged**